pyanaconda/payload/dnfpayload.py

```diff
@@ -393,26 +393,26 @@ def _add_repo(self, ksrepo):
 
         log.debug("added repo: '%s' - %s", ksrepo.name, url or mirrorlist or metalink)
 
-    def _fetch_md(self, repo):
+    def _fetch_md(self, reponame):
```

Comment:
Please change `reponame` to `repo_name`.

Reply:
Done (what's 13 months between friends)